src/test/config.sol

```diff
@@ -1607,5 +1607,35 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["WSTETH-B"] = CollateralValues({
```

Comment:
Maybe group this under `WSTETH-A`. Not sure if there is a hard startdard there but seems it is happening with other collaterals.

Reply:
Done.